README.md

```diff
@@ -218,8 +218,35 @@ public class MyKafkaStreams extends KafkaStreamsStarter {
         return "dlqTopic";
     }
 }
+```   
+
+Additionally, you may bring your own uncaught exception handler if you choose to do so. This provides an ability to
```

Comment:
Add this section in a dedicated "#### Uncaught Exception Handler" after "#### Avro Schema": https://github.com/michelin/kstreamplify/blob/main/README.md#avro-schema.

Think about updating the table of contents